src/api/index.js

```diff
@@ -25,15 +25,24 @@ export function getHosts() {
 
 export function downloadPlaybook(selectedIds) {
   return new Promise((resolve) => {
-    const tab = downloadPlaybooksParamCreator({
-      selectedRemediations: selectedIds,
-      options: { baseURL: API_BASE },
-    }).then((res) => {
-      const { search, pathname } = res.urlObj;
-      const url = `${API_BASE}${pathname}${search}`;
-      window.location.assign(url);
-      return true;
-    });
+    const tab =
+      selectedIds?.length > 1
+        ? downloadPlaybooksParamCreator({
+            selectedRemediations: selectedIds,
+            options: { baseURL: API_BASE },
+          }).then((res) => {
+            const { search, pathname } = res.urlObj;
+            const url = `${API_BASE}${pathname}${search}`;
+            window.location.assign(url);
+            return true;
+          })
+        : //TODO: Once JS Client is updated, replace with JS Client downloadPLaybooksParamCreator
```

Comment:
The JS clients are not updated automatically, so unless someone updates the remediations client no one will. ;)

Reply:
Does the OpenAPI definition still need updating? Otherwise it would just require running a task in the js clients repo and submit a PR with the updates?